lib/ptor.ts

```diff
@@ -23,7 +23,7 @@ export class Ptor {
   By: ProtractorBy;
```

Comment:
nit: up on line 11, prefer `const webdriver = ...`.  Or possibly just remove the declaration entirely if you can address my other nit in this file